script/run_hadolint.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash -e
+
+REPO_ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")" > /dev/null 2>&1 && pwd)/../"
+ALL_FILES=$(find $REPO_ROOT -name Dockerfile)
+
+echo "Stopping lingering hadolint containers if any"
+docker stop my_linter || true
+docker rm my_linter || true
+
+echo "Starting linter locally"
+docker run -di --name my_linter hadolint/hadolint
+
+echo "Scanning each Dockerfile"
+echo "$ALL_FILES" | while read line 
+do
```

Comment:
Couldn't you have combined the loop and the find to make
```bash
for line in $(find "$REPO_ROOT" -name Dockerfile)
do
```
instead of using echo read